src/main/resources/db/migration/V1__create_auth_table.sql

```diff
@@ -0,0 +1,6 @@
+-- V1__create_user_table.sql
+CREATE TABLE auth (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    password VARCHAR(255) NOT NULL
```

Comment:
create database table name as user_authentication for clarity, which indicates it stores user authentication-related information.
It is also a good practice to add 'created_at', 'updated_at' column to track when a user was created and last modified. These timestamps can be useful for logging or tracking user activity.

Reply:
resolved